src/import/generic/memory/lib/utils/conversions.H

```diff
@@ -88,7 +88,9 @@ static const std::vector<std::pair<uint64_t, uint64_t>> FREQ_TO_CLOCK_PERIOD =
 {
     {DIMM_SPEED_1600, 1250}, // DDR4
     {DIMM_SPEED_1866, 1071}, // DDR4
+    {DIMM_SPEED_1866, 1072}, // DDR4 - rounding error.
     {DIMM_SPEED_2133,  937}, // DDR4
+    {DIMM_SPEED_2133,  938}, // DDR4 - rounding error.
```

Comment:
I'm assuming that the rounding errors are due to calculations? where are the errors coming from (wasn't sure if it was bugged code or floating point math having issues on systems)?

Reply:
The error are coming from the memory manufacturer. The incorrect value is programmed into the part.

Reply:
Understood, thanks for the response.